addons/medical_gui/functions/fnc_updateInjuryList.sqf

```diff
@@ -0,0 +1,148 @@
+    };
+};
+
+// Indicate if a tourniquet is applied
+if (HAS_TOURNIQUET_APPLIED_ON(_target,_selectionN)) then {
+    _entries pushBack [localize LSTRING(Status_Tourniquet_Applied), [0.77, 0.51, 0.08, 1]];
+};
+
+// Indicate the amount of pain the unit is in
+if ([_target] call EFUNC(common,isAwake)) then {
+    private _pain = GET_PAIN_PERCEIVED(_target);
+    if (_pain > 0) then {
+        private _painText = switch (true) do {
+            case (_pain > 0.5): {
+                ELSTRING(medical_treatment,Status_SeverePain);
+            };
+            case (_pain > 0.1): {
+                ELSTRING(medical_treatment,Status_Pain);
+            };
+            default {
+                ELSTRING(medical_treatment,Status_MildPain);
+            };
+        };
+        _entries pushBack [localize _painText, [1, 1, 1, 1]];
+    };
+};
+
+// Show receiving IV volume remaining
+private _totalIvVolume = 0;
+{
+    _x params ["_volumeRemaining"];
+    _totalIvVolume = _totalIvVolume + _volumeRemaining;
+} forEach (_target getVariable [QEGVAR(medical,ivBags), []]);
+
+if (_totalIvVolume >= 1) then {
+    _entries pushBack [format [localize ELSTRING(medical_treatment,receivingIvVolume), floor _totalIvVolume], [1, 1, 1, 1]];
+};
+
+// Add entries for open, bandaged, and stitched wounds
+private _woundEntries = [];
+
+private _fnc_getWoundDescription = {
+    private _className = EGVAR(medical_damage,woundsData) select _woundClassID select 6;
+    private _suffix = ["Minor", "Medium", "Large"] select _category;
+    private _woundName = localize format [ELSTRING(medical_damage,%1_%2), _className, _suffix];
+    if (_amountOf >= 1) then {
+        format ["%1x %2", ceil _amountOf, _woundName];
+    } else {
+        format ["Partial %1", _woundName];
+    };
+};
+
+{
+    _x params ["", "_woundClassID", "_bodyPartN", "_amountOf", "", "", "_category"];
+    if (_selectionN == _bodyPartN) then {
+        if (_amountOf > 0) then {
+            _woundEntries pushBack [call _fnc_getWoundDescription, [1, 1, 1, 1]];
+        } else {
+            if !(EGVAR(medical_treatment,advancedBandages) && {EGVAR(medical_treatment,woundReopening)}) then {
+                _woundEntries pushBack [format ["[B] %1", call _fnc_getWoundDescription], [0.7, 0.7, 0.7, 1]];
+            };
+        };
+    };
+} forEach (_target getVariable [QEGVAR(medical,openWounds), []]);
+
+{
+    _x params ["", "_woundClassID", "_bodyPartN", "_amountOf", "", "", "_category"];
+    if (_selectionN == _bodyPartN && {_amountOf > 0}) then {
+        _woundEntries pushBack [format ["[B] %1", call _fnc_getWoundDescription], [0.88, 0.7, 0.65, 1]];
+    };
+} forEach (_target getVariable [QEGVAR(medical,bandagedWounds), []]);
+
+{
+    _x params ["", "_woundClassID", "_bodyPartN", "_amountOf", "", "", "_category"];
+    if (_selectionN == _bodyPartN && {_amountOf > 0}) then {
+        _woundEntries pushBack [format ["[S] %1", call _fnc_getWoundDescription], [0.7, 0.7, 0.7, 1]];
```

Comment:
Wha is the `[S]` for?

Reply:
`[S]` was there previously, I assume to indicate a stitched wound.